omod/src/main/webapp/pages/login.gsp

```diff
@@ -19,67 +19,73 @@
 
 ${ ui.includeFragment("referenceapplication", "infoAndErrorMessages") }
 
+<% if(showSessionLocations) { %>
 <script type="text/javascript">
     jQuery(function() {
-    	updateSelectedOption = function() {
-	        jQuery('#sessionLocation li').removeClass('selected');
-	        
-			var sessionLocationVal = jQuery('#sessionLocationInput').val();
-	        if(sessionLocationVal != null && sessionLocationVal != "" && sessionLocationVal != 0){
-	            jQuery('#sessionLocation li[value|=' + sessionLocationVal + ']').addClass('selected');
-	        }
-    	};
-    
+        updateSelectedOption = function() {
+            jQuery('#sessionLocation li').removeClass('selected');
+
+            var sessionLocationVal = jQuery('#sessionLocationInput').val();
+            if(sessionLocationVal != null && sessionLocationVal != "" && sessionLocationVal != 0){
+                jQuery('#sessionLocation li[value|=' + sessionLocationVal + ']').addClass('selected');
+            }
+        };
+
         updateSelectedOption();
 
         jQuery('#sessionLocation li').click( function() {
             jQuery('#sessionLocationInput').val(jQuery(this).attr("value"));
             updateSelectedOption();
         });
-		jQuery('#sessionLocation li').focus( function() {
+        jQuery('#sessionLocation li').focus( function() {
             jQuery('#sessionLocationInput').val(jQuery(this).attr("value"));
             updateSelectedOption();
         });
-		
-		// If <Enter> Key is pressed, submit the form
-		jQuery('#sessionLocation').keyup(function (e) {
-    		var key = e.which || e.keyCode;
-    		if (key === 13) {
-      			jQuery('#login-form').submit();
-    		}
-		});
-		var  listItem = Array.from(jQuery('#sessionLocation li'));
-		for (var i in  listItem){
-			 listItem[i].setAttribute('data-key', i);
-			 listItem[i].addEventListener('keyup', function (event){
-				var keyCode = event.which || event.keyCode;
-				switch (keyCode) {
-					case 37: // move left
```

Comment:
Too many indention changes. Can we do or fix the indention using another pull request? It gets hard to see what actually changed.

Reply:
I have done the refactor for those indentions. It occured automatically since I done it through the IntelliJ. Could be better to go through this :smile:

Reply:
I have updated all the indentions changes

Reply:
Isn't this another indention?

Reply:
Extremely sorry @dkayiwa  It happened automatically with my IDE. I just made it manually now. Please have a look again